integrations/rendertron/compose-tests.yml

```diff
@@ -0,0 +1,21 @@
+# IMPORTANT: The version must match the version of docker-compose.yml
+---
+version: "3"
+
+# All additional integrations should be added following this format only.
+
+services:
+  rendertron:
+    build:
+      context: ../integrations/rendertron
+      dockerfile: Dockerfile
+    container_name: intelowl_rendertron
+    restart: unless-stopped
+    expose:
+      - "4006"
+    env_file:
+      - env_file_integrations
+    volumes:
```

Comment:
Probably not required here since renderton does not the log the way other containers do.